mfr/extensions/codepygments/static/css/default.css

```diff
@@ -60,4 +60,13 @@
 .highlight .vc { color: #19177C } /* Name.Variable.Class */
 .highlight .vg { color: #19177C } /* Name.Variable.Global */
 .highlight .vi { color: #19177C } /* Name.Variable.Instance */
-.highlight .il { color: #666666 } /* Literal.Number.Integer.Long */
\ No newline at end of file
+.highlight .il { color: #666666 } /* Literal.Number.Integer.Long */
+
+/* CSS only for Firefox */
+@-moz-document url-prefix() {
+  .highlight > pre {
+      white-space: pre-wrap;       /*  css-3 */
+      white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+      word-break: break-all;
+  }
+}
```

Comment:
Incredibly minor, but generally try to make sure that all files end with a newline. (some IDEs have a setting to check this automatically)

It won't make or break the OSF, but every now and then a tool will work oddly if there's no newline at the end of a file. (hence why github shows that warning icon) See: http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline